api/docs/v2/storefront/index.yaml

```diff
@@ -161,10 +161,10 @@ paths:
                       type: string
                       example: john@snow.org
                     bill_address_id:
-                      type: integer
```

Comment:
@pmilewski we also need to update request specs to test against strings, not integers here: https://github.com/spree/spree/blob/main/api/spec/requests/spree/api/v2/storefront/account_spec.rb#L91